.github/workflows/docker.yml

```diff
@@ -18,15 +18,19 @@ jobs:
 
   pytest-api:
     runs-on: ubuntu-latest
+    env:
+        # Port for web container
+        # See docker-compose.yml
+        PORT: 8002
     steps:
       - uses: actions/checkout@v2
       - name: Build & run docker
-        run: PORT=8002 docker-compose up -d --build
+        run: docker-compose up -d --build
       - name: Ping server
-        run: curl http://localhost:8002/docs
+        run: wget --spider --tries=12 http://localhost:$PORT/docs
```

Comment:
Could you remind me the reason of this change? (not familiar with the spider option, the "tries" flag seems explicit)

Reply:
spider does only an 'exist' check and avoid downloading anything

Reply:
@frgfm Also, `curl` has an option `--retry-all-errors` to retry in any case but this option is only available in latest version of `curl` which can't be (easily) installed on `Ubuntu 20.04`. So it may still fail for various reasons that we can't catch. However, `wget` comes out of the box with a "retry on all errors".

Reply:
@frgfm Concerning your comment on version, it's a good remark. I didn't changed anything on my side and it didn't install the latest version of fastapi :thinking: